from geolab.exceptions import PIValueError, PSDValueError
from geolab.soil_classifier import Soil
